docs/references/room.md

```diff
@@ -2,624 +2,242 @@
-| :--- | :--- | :--- |
-| id | `string` | Get Room id. This function is depending on the Puppet Implementation, see [puppet-compatible-table](https://github.com/wechaty/wechaty/wiki/Puppet#3-puppet-compatible-table) |
-
-* [Room](room.md#Room)
-  * _instance_
-    * [.sync\(\)](room.md#Room+sync) ⇒ `Promise <void>`
-    * [.say\(textOrContactOrFileOrUrl, ...mentionList\)](room.md#Room+say) ⇒ `Promise <void>`
-    * [.on\(event, listener\)](room.md#Room+on) ⇒ `Room`
-    * [.add\(contact\)](room.md#Room+add) ⇒ `Promise <void>`
-    * [.del\(contact\)](room.md#Room+del) ⇒ `Promise <void>`
-    * [.quit\(\)](room.md#Room+quit) ⇒ `Promise <void>`
-    * [.topic\(\[newTopic\]\)](room.md#Room+topic) ⇒ `Promise <void | string>`
-    * [.announce\(\[text\]\)](room.md#Room+announce) ⇒ `Promise <void | string>`
-    * [.qrcode\(\)](room.md#Room+qrcode) ⇒ `Promise <string>`
-    * [.alias\(contact\)](room.md#Room+alias) ⇒ `Promise <null | string>`
-    * [.has\(contact\)](room.md#Room+has) ⇒ `Promise <boolean>`
-    * [.memberAll\(\[query\]\)](room.md#Room+memberAll) ⇒ `Promise <Contact []>`
-    * [.member\(queryArg\)](room.md#Room+member) ⇒ `Promise <Contact | null>`
-    * [.owner\(\)](room.md#Room+owner) ⇒ `Contact` \| `null`
-    * [.avatar\(\)](room.md#room-owner-contact-or-null) ⇒ `Promise <FileBox>`
-  * _static_
-    * [.create\(contactList, \[topic\]\)](room.md#Room.create) ⇒ `Promise <Room>`
-    * [.findAll\(\[query\]\)](room.md#Room.findAll) ⇒ `Promise <Room []>`
-    * [.find\(query\)](room.md#Room.find) ⇒ `Promise <Room | null>`
-
-### room.sync\(\) ⇒ `Promise <void>`
-
-Force reload data for Room, Sync data from lowlevel API again.
-
-**Kind**: instance method of [`Room`](room.md#Room) **Example**
-
-```javascript
-await room.sync()
+```python
+class Room(Accessory[RoomPayload])
 ```
 
-### room.say\(textOrContactOrFileOrUrlLinkOrMiniProgram, ...mentionList\) ⇒ `Promise <void>`
-
-Send message inside Room, if set mentionList, wechaty will mention the contact list as well.
-
-> Tips: This function is depending on the Puppet Implementation, see [puppet-compatible-table](https://github.com/wechaty/wechaty/wiki/Puppet#3-puppet-compatible-table)
-
-**Kind**: instance method of [`Room`](room.md#Room)
-
-| Param | Type | Description |
-| :--- | :--- | :--- |
-| textOrContactOrFileOrUrlLinkOrMiniProgram | `string` \| `Contact` \| `FileBox` \| `UrlLink` \| `MiniProgram` | Send `text`, `media file` or `link` inside Room.   You can use [FileBox](https://www.npmjs.com/package/file-box) to send file |
-| ...mentionList | `Contact []` | Send content inside Room, and mention @contact list. |
-
-#### Exampl
-
-```javascript
-import { FileBox }  from 'file-box'
-import {
-  Wechaty,
-  UrlLink,
-  MiniProgram,
-}  from 'wechaty'
-
-const bot = new Wechaty()
-await bot.start()
-// after logged in...
-const room = await bot.Room.find({topic: 'wechaty'})
-
-// 1. Send text inside Room
-
-await room.say('Hello world!')
-
-// 2. Send media file inside Room
```

Comment:
Please translate source docs to chinese description and make it described it clearly.